src/ui/__tests__/ui_nav_link.js

```diff
@@ -0,0 +1,52 @@
+import expect from 'expect.js';
+
+import UiNavLink from '../ui_nav_link';
+
+describe('UiNavLink', () => {
```

Comment:
This class has a couple other wrinkles to its logic that I think are worth testing / document via tests:

* `uiExports.urlBasePath` defaults to an empty string if it's falsy, which affects the UiNavLink instance's `url` prop.
* `order` defaults to 0.
* `linkToLastSubUrl` will be false _only_ if `spec.linkToLastSubUrl` is false, but will default to true for any other value (including falsy ones!).